### ISSUE TITLE
Cannot mint on Safari

### ISSUE THREAD

**Issue by uloco** [hicetnunc2000/hicetnunc#200]
Whenever I try minting an OBJKT in Safari, I get an unhandled promise rejection (in the console) and nothing happens on the page.

**Comment by crzypatchwork** [comment]
we are reaching a stable version for all interactions with the blockchain :) we might revert the UI to it's early version to make minting as simple as possible. as well with other interactions with the dApp.

**Comment by andrevenancio** [comment]
> Whenever I try minting an OBJKT in Safari, I get an unhandled promise rejection (in the console) and nothing happens on the page.

Is there more information you can provide? what asset were you uploading? can you share it here so I can test?

**Comment by uloco** [comment]
I minted it successfully afterwards in Chrome because someone wanted to buy that piece. It's already sold. But it was not a particularly big asset. I selected it from the photos tab in the explorer view, because it is saved in my photos library there. ? It is a PNG file, 1.2MB of size, 1640 × 2360, the original file name is IMG_0069.PNG. I also set a caption in photos: "you tear me apart".
Here is the link to the successfully minted objkt: https://www.hicetnunc.xyz/objkt/8677
